Sparkle/Autoupdate/Autoupdate.m

```diff
@@ -208,10 +208,12 @@ - (void)cleanupAndTerminateWithPathToRelaunch:(NSString *)relaunchPath
         // The only remedy I've been able to find is waiting an arbitrary delay before exiting our application
         
         // Don't use -launchApplication: because we may not be launching an application. Eg: it could be a system prefpane
+#pragma GCC diagnostic push
```

Comment:
So, what happened was, for some reason because of the complex settings tree we have and they have, the warning suppression flags weren't being picked up no matter what I tried. So I opted to shut up its whining at each code site. Sorry. None of the other submods required this much convincing the shut the hell up.